UI/payments/payment2.html

```diff
@@ -162,7 +162,7 @@
                                  data-dojo-type="dijit/form/Button">
                                 [% column.text -%]
                                 <script type="dojo/on" data-dojo-event="click">
-                                    require(["dojo/domReady!"], function(){
+                                    require(["dojo/ready"], function(){
```

Comment:
Didn't we replace the entire `<script>...</script>` block to replace it with the "ToggleIncludeButton"?

Reply:
btw, neither `ready` nor `domReady` is applicable at this point: the dom will have long finished loading when this code is triggered.

Reply:
Fixing PR missing on master. Solved